database/schema.sql

```diff
@@ -29,10 +26,13 @@ CREATE TABLE public.users (
   id VARCHAR PRIMARY KEY NOT NULL UNIQUE CHECK (id <> ''),
   nonce UUID UNIQUE,
   avatar_url VARCHAR CHECK (avatar_url <> ''),
-  cover_photo_url VARCHAR CHECK (cover_photo_url <> ''),
   email VARCHAR UNIQUE CHECK (email <> ''),
   username VARCHAR UNIQUE CHECK (username <> ''),
-  preferences VARCHAR UNIQUE CHECK (preferences <> ''),
+  name VARCHAR CHECK (name <> ''),
+  description VARCHAR CHECK (description <> ''),
+  title VARCHAR CHECK (title <> ''),
+  status VARCHAR CHECK (status <> ''),
+  location VARCHAR CHECK (location <> ''),
```

Comment:
I've already seen this changes from the previous pull request. Looks like you might have to pull in the latest changes coming from the main repo if you're on your fork. You'll have to sync it up together.